packages/flutter/lib/src/material/radio.dart

```diff
@@ -447,7 +447,9 @@ class _RadioState<T> extends State<Radio<T>> with TickerProviderStateMixin, Togg
               value: widget.value,
               groupValue: widget.groupValue,
               onChanged: widget.onChanged,
-              mouseCursor: widget.mouseCursor,
+              mouseCursor: WidgetStateProperty.all(widget.mouseCursor
+                ?? (kIsWeb && widget.onChanged != null
+                  ? SystemMouseCursors.click : SystemMouseCursors.basic)),
```

Comment:
Is this duplicating CupertinoRadio._defaultMouseCursor? Could you make that public and use it here?

Reply:
I added a static method `defaultMouseCursor` in `CupertinoRadio` that I just call here to get the default mouse cursor.